editoast/src/views/work_schedules.rs

```diff
@@ -14,7 +14,7 @@ use std::result::Result as StdResult;
 use thiserror::Error;
 use utoipa::ToSchema;
 
-use crate::core::pathfinding::TrackRange as CoreTrackRange;
+use crate::core::pathfinding::DirectionalTrackRange as CoreTrackRange;
```

Comment:
```suggestion
use crate::core::pathfinding::DirectionalTrackRange;
```

nitpick: the aliasing is there to avoid conflict with `editoast_schemas::TrackRange`. Now that the names naturally don't conflict anymore the `as` becomes superfluous.

Reply:
Nah I believe the names will conflict again... The types were leaked to the front, so the renaming becomes a bit involved and better reserved for another PR

Reply:
I reverted the renaming of `core::pathfinding::TrackRange` to `DirectionalTrackRange` due to the build errors it caused in `front` (after updating the api and generating the typescript types succeeded). Currently the aliasing is still needed to distinguish `core` track ranges and `schemas` track ranges. Should I resolve this conversation ?

Reply:
> I reverted the renaming of core::pathfinding::TrackRange to DirectionalTrackRange due to the build errors it caused in front

Too bad. Resolving this then.